crates/storage/store/storage.rs

```diff
@@ -476,6 +476,10 @@ impl Store {
     ) -> Result<H256, StoreError> {
         let mut genesis_state_trie = self.engine.open_state_trie(*EMPTY_TRIE_HASH);
         for (address, account) in genesis_accounts {
+            println!(
+                "Inserting account: {:#x} with balance: {}",
+                address, account.balance
+            );
```

Comment:
This might be better as a debug log.

Reply:
I should probably remove it , I added it for debugging, thanks!